board/views.py

```diff
@@ -28,38 +28,31 @@ def __init__(self):
         self.platforms = res_platforms
 
 
-class BaseGameView:
-    def __init__(self, user):
-        self.user = user
-
-    def _get_context(self, game_id, is_favourite):
-        if Game.is_exist(game_id):
-            game = Game(game_id)
-            tweets = Tweet.get_tweets(game_id)
-        else:
-            game = GameAPI(game_id)
-            tweet_ids = twitter_wrapper.get_tweets_by_string(game.slug)
-            if tweet_ids:
-                tweets = [TweetAPI(tweet_id) for tweet_id in tweet_ids]
-            else:
-                tweets = None
+class DetailView(View):
+    def get(self, request, game_id):
+        game = self._get_game(game_id)
+        tweets = self._get_tweets(game.slug)
 
+        is_favourite = Favourite.objects.filter(game_id=self.kwargs['game_id']).exists()
```

Comment:
Is that correct that if any user will add game to favourites, all users would see the game as favourite?